pkg/extensions/sync/utils.go

```diff
@@ -133,7 +133,7 @@ func getHTTPClient(regCfg *RegistryConfig, credentials Credentials, log log.Logg
 	}
 
 	// nolint: gosec
-	if regCfg.TLSVerify != nil && !*regCfg.TLSVerify {
+	if regCfg.TLSVerify != nil && !*regCfg.TLSVerify && strings.HasPrefix(regCfg.URL, "https") {
```

Comment:
What if "httpsabc.weirdprefix.com"?
Would actually parse the uri and then .Scheme

Reply:
Ok, I'll use an url parser